docker-compose.yml

```diff
@@ -8,7 +8,7 @@ services:
     image: ruby:2.6
     tty: true
     stdin_open: true
```

Comment:
I suppose these two lines are there to access the container through the terminal?

Reply:
Yes! :)

Reply:
If this file is also used to run the test we could remove these two lines and add them like flags in the command line to launch it no? like --tty and -i? Not sure it will work

Reply:
I don't know if this flags exists actually, but they don't do any difference if you prefer running the commands like this: `docker-compose run --rm package bundle exec rspec`